internal/runtime/local/local.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022-2023 The Inspektor Gadget authors
```

Comment:
How about merging this and run.go in the same file? (I have switched N times between the two while studying the code :D).